### PR TITLE
Tickets/dm 30034

### PR DESCRIPTION
**Pull request by pcortes80**
Push the last changes in LabVIEW TcpServer code. Please review them.

### CODE REVIEW

**Review by teweitsai — CHANGES_REQUESTED** [review]
I put the review in the JIRA ticket. Let us discuss the details in tomorrow.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ This module should be compatable with the Python TCP/IP client.
 The python scripts are in `python/` directory.
 You can follow [here](python/README.md) for the details.
 
+## LabVIEW Componente Simulator
+
+The LabVIEW Component Simulator code is in `scr/` directory.
+You can follow [here](src/README.md) for detailes.
```
Comment (teweitsai) [inline README.md:19]: detailes --> details

File: src/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+### Build the Executable
+
+The LabVIEW professional version is needed to build the executable. 
+You can build the executable from within LabVIEW or from the command line, as you prefer:
+
+- From within LabVIEW: select the build all under Build Specifications of tcpip.lvproj. The ComponentSimulator executable will be in the `build/` directory.
+The output log is in the build/buildComponentSimulator.txt.
+- From the command line:
+```sh
```
Comment (teweitsai) [inline src/README.md:20]: I think there should be a new line before the code block. You could check the syntax by the vscode (with the plugin). There is the similar problems in other places as well.

File: src/README.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+```
+
+### Start Component Simulator
+
+From terminal:
+
+Go to the `build/` directory:
+```sh
+cd  ts_labview_tcp_json/build
+```
```
Comment (teweitsai) [inline src/README.md:31]: Add the new lines before and after the code block.

File: src/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Execute Component Simulator:
+
+```sh
+./ComponentSimulator
+```
+
+Check:
+- IP = 127.0.0.1
```
Comment (teweitsai) [inline src/README.md:39]: There should be a new line before this.

File: src/README.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+- Port = 8888
+
+Note: this configuration cannot be modify in runtime.
```
Comment (teweitsai) [inline src/README.md:42]: modify --> modified

File: src/README.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+
+### Start TCP Client
+
+For the use of TCP Python Client to send commands to TCP Server, you can follow [here](python/README.md).
```
Comment (teweitsai) [inline src/README.md:46]: This hyperlink does not work.

File: src/README.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+### Send Command Status to TCP Client
+
+If you want to send the Command Status to the TCP Cliente, do:
```
Comment (teweitsai) [inline src/README.md:50]: Cliente --> Client

**Review by teweitsai — APPROVED** [review]
This PR looks good to me. I put some comments in PR and another `reviewAgain.txt` in JIRA ticket. Please help to update the code before the merge. Thanks!

File: README.md (hunk 1 above)
Comment (pcortes80) [reply README.md:19]: Done.

File: src/README.md (hunk 2 above)
Comment (pcortes80) [reply src/README.md:20]: Done.

File: src/README.md (hunk 3 above)
Comment (pcortes80) [reply src/README.md:31]: Done.

File: src/README.md (hunk 4 above)
Comment (pcortes80) [reply src/README.md:39]: Done.

File: src/README.md (hunk 5 above)
Comment (pcortes80) [reply src/README.md:42]: Done.

File: src/README.md (hunk 6 above)
Comment (pcortes80) [reply src/README.md:46]: Done. Now the hyperlink works.

File: src/README.md (hunk 7 above)
Comment (pcortes80) [reply src/README.md:50]: Done.

**Comment by pcortes80** [comment]
Fix all comments in reviewAgain.txt.

**Pull request merged**